lib/OpenQA/Utils.pm

```diff
@@ -682,5 +683,36 @@ sub mark_job_linked {
     }
 }
 
+sub detect_current_version($) {
+    my ($path) = @_;
+    # Get application version
+    my $current_version = undef;
+
+    if (-e $path . '/public/Changelog') {
```

Comment:
You could use `Mojo::File` here (to make path handling portable), which `Mojo::Home` just so happens to be a subclass of.
```perl
use Mojo::File 'path';
...
my $changelog = path($path, 'public', 'Changelog');
if (-e $changelog) {
...
```